scripts/ynh_package_version

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+read_json () {
+    sudo python3 -c "import sys, json;print(json.load(open('$1'))['$2'])"
+}
+
+read_manifest () {
+    if [ -f '../manifest.json' ] ; then
+        read_json '../manifest.json' "$1"
+    else
+        read_json '../settings/manifest.json' "$1"
+    fi
+}
+abort_if_up_to_date () {
+    version=$(read_json "/etc/yunohost/apps/$YNH_APP_INSTANCE_NAME/manifest.json" 'version' 2> /dev/null || echo '20160501-7')
```

Comment:
Not sure what that `|| echo '20160501-7'` is about ? Copypasta from some other app ?

Reply:
Yes it's a copy paste from wekan, I've never noticed that before :/

It's from https://github.com/YunoHost-Apps/wekan_ynh/blob/6447c003126ddefbae478427b40bb760a5a9ec9d/scripts/_future.sh#L40